packages/core/src/lib/collapse.svelte

```diff
@@ -0,0 +1,68 @@
+<svelte:options immutable />
+
+<script lang="ts">
+import cx from 'classnames';
+import { createEventDispatcher } from 'svelte';
+import { Icon } from '$lib';
+
+type Variants = 'default' | 'minimal';
+
+export let title = '';
+export let open = false;
+export let variant: Variants = 'default';
+
+const dispatch = createEventDispatcher();
+
+const handleClick = () => {
+  dispatch('toggle', { isOpen: !open });
+};
+</script>
+
+<div class="relative w-full">
+  <div
+    role="button"
+    aria-label="Toggle Content"
+    tabindex="0"
+    class="{cx(
+      'w-full py-2 px-4 flex flex-reverse items-center justify-between text-default cursor-pointer',
+      {
+        'border border-light bg-white': variant === 'default',
+      }
+    )},"
+    on:click={handleClick}
+    on:keyup|preventDefault={handleClick}
+  >
+    <div class="flex flex-wrap gap-x-3 gap-y-1 items-center">
+      {#if title}
+        <h2 class="m-0 text-sm">{title}</h2>
+      {/if}
+
+      <slot name="title" />
+    </div>
+
+    <div class="h-full flex items-center gap-3">
+      <slot name="header" />
+
+      <div
+      class={cx(
```

Comment:
I would expect this to work/rotate my icon, but it's not.  Do I need to update the icon svelte component to accept class props?